contrib/docker/kwild.debug.dockerfile

```diff
@@ -15,6 +15,8 @@ RUN chmod 777 /var/run/kwil
 
 COPY . .
 
+RUN rm -f go.work && go work init . ./core ./test
```

Comment:
I did both(ci and docker) when doing KGW build, but I never really dig too much in this .
Now I think of it, isn't the path in the go.work are relative? shouldn't matter unless we use abs path right?

Reply:
ah, nvm, the go version is different